packages/material-ui/src/InputAdornment/InputAdornment.js

```diff
@@ -17,7 +17,7 @@ const overridesResolver = (props, styles) => {
     {
       ...styles[`position${capitalize(styleProps.position)}`],
       ...(styleProps.disablePointerEvents === true && styles.disablePointerEvents),
-      ...(styleProps.variant === 'filled' && styles.filled),
+      ...styles[styleProps.variant],
```

Comment:
@mnajdova I think this is simpler and also fix the bug.

Reply:
Agree